### ISSUE TITLE
Hystrix showing multiple daemon thread

### ISSUE THREAD

**Issue by mohan-mishra** [Netflix/Hystrix#1349]
@mattrjacobs @benjchristensen  While debugging, I am able to find the below thread dump in eclipse.
The used threads are not getting terminated resulting in load on server
```
Daemon Thread [HystrixTimer-3] (Running)	
Daemon Thread [hystrix-BUS_BUSINDIA-3] (Running)	
Daemon Thread [HystrixTimer-4] (Running)	
Daemon Thread [hystrix-BUS_BUSINDIA-4] (Running)	
Daemon Thread [RxComputationScheduler-2] (Running)	
Daemon Thread [hystrix-BUS_BUSINDIA-5] (Running)	
Daemon Thread [hystrix-BUS_BUSINDIA-6] (Running)	
Daemon Thread [hystrix-BUS_BUSINDIA-7] (Running)	
Daemon Thread [hystrix-BUS_BUSINDIA-8] (Running)	
Daemon Thread [hystrix-BUS_BUSINDIA-9] (Running)	
Daemon Thread [hystrix-BUS_BUSINDIA-10] (Running)	
Daemon Thread [hystrix-BUS_BUSINDIA-11] (Running)	
```

**Comment by mohan-mishra** [comment]
Also the daemon thread count is getting increase each time I am executing a hystrix command

**Comment by mukteshkrmishra** [comment]
I was wondering what is this BUSINDIA thread is ?

**Comment by mohan-mishra** [comment]
This is the name of my command group

**Comment by mukteshkrmishra** [comment]
Ideally this should be be the idea behavior, however matt or ben can comment on it further.

**Comment by mohan-mishra** [comment]
@mukteshkrmishra  What acc to you is ideal behaviour? Creation of daemon threads or termination of threads once their work is done

**Comment by mattrjacobs** [comment]
I think this is a duplicate of #1242.  Is that correct?

**Comment by mohan-mishra** [comment]
yes.
